clingo/README.md

```diff
@@ -12,4 +12,36 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+**TODO: Add instructions for usage**
```

Comment:
If we're making changes to the README, it would be ideal to have this follow the [readme template here](https://github.com/habitat-sh/core-plans/blob/master/README_TEMPLATE_FOR_PLANS.md).

Reply:
Done.